routes/api.php

```diff
@@ -19,6 +20,8 @@
         return $optionswatchModel::getCachedSwatchValues();
     });
 
+    Route::get('cart/order/{quoteIdMaskOrCustomerToken}', CartOrderController::class);
```

Comment:
```suggestion
    Route::get('order/{quoteIdMaskOrCustomerToken}', CartOrderController::class);
```

Why the `cart` prefix?